diff --git a/policy/modules/system/udev.fc b/policy/modules/system/udev.fc
index 491bb23..98f71dc 100644
--- a/policy/modules/system/udev.fc
+++ b/policy/modules/system/udev.fc
@@ -10,7 +10,7 @@
 /etc/udev/scripts/.+ --	gen_context(system_u:object_r:udev_helper_exec_t,s0)
 
 /lib/udev/udev-acl --	gen_context(system_u:object_r:udev_exec_t,s0)
-/lib/udev/udevd    --	gen_context(system_u:object_r:udev_exec_t,s0)
+/lib/udev/udevd    --	gen_context(system_u:object_r:bin_t,s0)
 
 ifdef(`distro_debian',`
 /lib/udev/create_static_nodes -- gen_context(system_u:object_r:udev_exec_t,s0)
