diff --git a/xfsettingsd/main.c b/xfsettingsd/main.c
index 55c4183..c94638a 100644
--- a/xfsettingsd/main.c
+++ b/xfsettingsd/main.c
@@ -271,14 +271,8 @@ main (gint argc, gchar **argv)
 
     /* create the sub daemons */
 #ifdef HAVE_XRANDR
-    displays_helper = g_object_new (XFCE_TYPE_DISPLAYS_HELPER, NULL);
 #endif
-    pointer_helper = g_object_new (XFCE_TYPE_POINTERS_HELPER, NULL);
-    keyboards_helper = g_object_new (XFCE_TYPE_KEYBOARDS_HELPER, NULL);
-    accessibility_helper = g_object_new (XFCE_TYPE_ACCESSIBILITY_HELPER, NULL);
     shortcuts_helper = g_object_new (XFCE_TYPE_KEYBOARD_SHORTCUTS_HELPER, NULL);
-    keyboard_layout_helper = g_object_new (XFCE_TYPE_KEYBOARD_LAYOUT_HELPER, NULL);
-    workspaces_helper = g_object_new (XFCE_TYPE_WORKSPACES_HELPER, NULL);
 
     if (g_getenv ("XFSETTINGSD_NO_CLIPBOARD") == NULL)
     {
@@ -312,14 +306,8 @@ main (gint argc, gchar **argv)
     /* release the sub daemons */
     g_object_unref (G_OBJECT (xsettings_helper));
 #ifdef HAVE_XRANDR
-    g_object_unref (G_OBJECT (displays_helper));
 #endif
-    g_object_unref (G_OBJECT (pointer_helper));
-    g_object_unref (G_OBJECT (keyboards_helper));
-    g_object_unref (G_OBJECT (accessibility_helper));
     g_object_unref (G_OBJECT (shortcuts_helper));
-    g_object_unref (G_OBJECT (keyboard_layout_helper));
-    g_object_unref (G_OBJECT (workspaces_helper));
 
     if (G_LIKELY (clipboard_daemon != NULL))
     {
